etc/kayobe/ansible/smartmon-tools.yml

```diff
@@ -49,3 +48,21 @@
         - smartmon
         - nvmemon
       become: yes
+
+    - name: Ensure the DWPD Ratings directory exists
+      file:
+        path: /opt/kayobe/etc/monitoring
+        state: directory
+        mode: '0755'
+      when: stackhpc_dwpd_ratings is defined
+      become: true
+
+    - name: Create a DWPD ratings file
+      copy:
+        content: |
```

Comment:
This could just be the raw file that you write out above when running the playbook?